scripts/ci/lib.sh

```diff
@@ -933,6 +933,23 @@ get_pr_details() {
 openshift_ci_mods() {
     info "BEGIN OpenShift CI mods"
 
+    #TODO(janisz): remove after https://github.com/openshift/release/pull/49962 is merged
+    if [ -e "/tmp/go/" ]; then
+        info "Go already exists"
+    else
+        info "Replace Go"
+        go version
+        whoami
+        GOLANG_VERSION=1.21.8
```

Comment:
Shouldn't we use here 1.21.7 as in the other places in this PR (and in the downstream)?
```suggestion
        GOLANG_VERSION=1.21.8
```

Reply:
Yeah, I thought about that (and @RTann asked too) having difference in bugfix version shouldn't be a big deal. 
https://github.com/stackrox/rox-ci-image/pull/199#pullrequestreview-1932767312